habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,246 @@
+
+# TODO: is there a better way to do this more generaly? Maybe pybind a cmake compiled DATA_DIR?
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_dir = os.path.abspath(os.path.join(dir_path, "../../data/")) + "/"
+
+# dict keyed by uids with download specs for various test assets and datasets
+data_sources = {
+    # "example-uid": {
+    #
+    # }
+    "habitat-test-scenes": {
+        "source": "http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip",
+        "package_name": "habitat-test-scenes.zip",
+        "unpack_to": data_dir + "../",
+        "root": data_dir + "scene_datasets/habitat-test-scenes/",
+        "version": "1.0",
+    },
+    "habitat-example-objects": {
+        "source": "http://dl.fbaipublicfiles.com/habitat/objects_v0.2.zip",
+        "package_name": "objects_v0.2.zip",
+        "unpack_to": data_dir + "objects/example_objects/",
+        "root": data_dir + "objects/example_objects/",
+        "version": "0.2",
+    },
+    "locobot-merged": {
+        "source": "http://dl.fbaipublicfiles.com/habitat/locobot_merged_v0.2.zip",
+        "package_name": "locobot_merged_v0.2.zip",
+        "unpack_to": data_dir + "objects/locobot_merged/",
+        "root": data_dir + "objects/locobot_merged/",
+        "version": "0.2",
+    },
+    "mp3d-test-scene": {
+        "source": "http://dl.fbaipublicfiles.com/habitat/mp3d_example.zip",
+        "package_name": "mp3d_example.zip",
+        "unpack_to": data_dir + "scene_datasets/mp3d_test/",
+        "root": data_dir + "scene_datasets/mp3d_test/",
+        "version": "1.0",
+    },
+    "coda-scene": {
+        "source": "'https://docs.google.com/uc?export=download&id=1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853'",
+        "package_name": "coda.zip",
+        "download_pre_args": "--no-check-certificate ",
+        "download_post_args": " -O " + data_dir + "coda.zip",
+        "unpack_to": data_dir + "scene_datasets/",
+        "root": data_dir + "scene_datasets/coda/",
+        "version": "1.0",
+    },
+}
+
+data_groups = {
+    "ci-test-assets": [
+        "habitat-test-scenes",
+        "habitat-example-objects",
+        "locobot-merged",
+        "mp3d-test-scene",
+        "coda-scene",
+    ]
+}
+
+# validatation: ids are unique between groups and sources
+for key in data_groups:
+    assert key not in data_sources, "Duplicate key: " + key
+
+
+def prompt_yes_no(message):
+    print("\n-------------------------")
+    print(message)
+    while True:
+        answer = input("(y|n): ")
+        if answer.lower() == "y":
+            return True
+        elif answer.lower() == "n":
+            return False
+        else:
+            print("Invalid answer...")
+
+
+def clean_data(uid):
+    if uid not in data_sources:
+        print("Data download failed, no datasource named " + uid)
+        return
+    print(
+        "Cleaning datasource " + uid + ' directory: "' + data_sources[uid]["root"] + '"'
+    )
+    os.system("rm -r " + data_sources[uid]["root"])
```

Comment:
Use shutil.rmtree instead here. Very bad to use rm like this. What could go wrong... 0_0